docs/Makefile

```diff
@@ -69,6 +69,13 @@ pre-publish:
 .PHONY: publish-check
 	python3 redirects.py publish-check
 
+.PHONY: check-links
+check-links:
+	make attributions.rst
+	# any warnings will dump to stderr, continue regardless to parse output.txt
+	sphinx-build -b linkcheck . build -a || true
+	! (cat build/output.txt | grep -n broken)
```

Comment:
If @dzhu were here, he'd chide you for using an extra cat and pipe instead of just 
```bash
grep -n broken < build/output.txt
```
and I would begrudgingly admit that he's right about that.

Reply:
In fact, no need for the `<`!